build.gradle

```diff
@@ -94,4 +94,10 @@ test {
 
 task wrapper(type: Wrapper) {
     gradleVersion = '2.3'
-}
\ No newline at end of file
+}
+
+task copyTestResources(type: Copy) {
```

Comment:
@benrudolph I needed this gradle task in order to make my tests work from IntelliJ http://stackoverflow.com/a/26033752